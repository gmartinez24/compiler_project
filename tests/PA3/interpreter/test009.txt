// Aum Palande
// Function calls with nested if statements
main

int x, y, z, a, b;

{
    a = call readInt();
    b = call readInt();
    z = -1;
    if (a > b) then
	x = call readInt();
	y = call readInt();
	if (x > y) then
	    z = 1;
	else 
	    z = 0;
	fi;
    fi;
    printInt(z);
}.