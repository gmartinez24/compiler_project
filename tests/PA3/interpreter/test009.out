int? int? int? int? 0
